denizen_scripts/survival/repo-link/meeseeks_shit/BawbShop.dsc

```diff
@@ -123,7 +123,7 @@ Meeseeks_Operator:
                 - narrate format:NPC Howdy!
                 - stop
         - else:
-            - define Action <[NbtData].unescaped.as_list.map_get[Action]>
+            - define Action <[NbtData].unescaped.as_map.get[Action]>
```

Comment:
This escaped data should be formatted properly as opposed to escaped where-ever it actually came from and unescaped here